### ISSUE TITLE
No GTask wrappers?

### ISSUE THREAD

**Issue by shutton** [gtk-rs/gio#304]
I notice that there's no wrapper code for GTask.  I was just wondering if that was a conscious decision, or simply work that hadn't been done yet, since it seems like a rather glaring omission.

GTask is (or could be) quite helpful in exporting Rust-based implementations to existing C/GObject-based environments that are already designed around async/finish APIs.

**Comment by GuillaumeGomez** [comment]
I think it's just something we weren't aware of. :)

**Comment by sdroege** [comment]
No, it's just something I didn't get to yet :) If you want to work on it, please go ahead!

**Comment by sdroege** [comment]
Let's keep it open so it's not forgotten :)
